### PR TITLE
Update port binding documentation

### PR DESCRIPTION
**Pull request by gaurav-dalvi**

### CODE REVIEW

File: net/extras/sample_aci_cfg.yml [hunk 1]
```diff
@@ -33,9 +33,23 @@
 #APIC_EPG_BRIDGE_DOMAIN: "not_specified"
 #APIC_CONTRACTS_UNRESTRICTED_MODE: "no"
 
-## Full path of the leaf nodes connected to your servers.
-## List each node on a new line as shown commented
+# Topology information can be given following 3 ways.
```
Comment (jojimt) [inline net/extras/sample_aci_cfg.yml]: Please replace with the below.

Provide server to ACI physical connectivity information below. This information is used for establishing EPG bindings. You can specify node level binding, path level binding, or a mix of both as shown in examples.

File: net/extras/sample_aci_cfg.yml [hunk 2]
```diff
@@ -33,9 +33,23 @@
...
+
+# Full path of leaves:
```
Comment (jojimt) [inline net/extras/sample_aci_cfg.yml]: Node level binding

File: net/extras/sample_aci_cfg.yml [hunk 3]
```diff
@@ -33,9 +33,23 @@
...
 #APIC_LEAF_NODES:
 #    - topology/pod-1/node-101
 #    - topology/pod-1/node-102
+
+## When you want port binding, you can use following topology syntax.
```
Comment (jojimt) [inline net/extras/sample_aci_cfg.yml]: Port level binding (aka path level binding)

File: net/extras/sample_aci_cfg.yml [hunk 4]
```diff
@@ -33,9 +33,23 @@
...
+#APIC_LEAF_NODES:
+#    - topology/pod-1/paths-101/pathep-[eth1/14]
+#    - topology/pod-1/paths-101/pathep-[eth1/15]
+## Here we are binding to only eth1/14 and eth1/15 ports of leaf 101 as opposed to full leaf level binding in earlier case.
+
+# You can give mix of both like this.
```
Comment (jojimt) [inline net/extras/sample_aci_cfg.yml]: Mix of nodes and ports.

**Comment by gaurav-dalvi** [comment]
@jojimt : Please take a look now.

**Pull request merged**